solidity/dashboard/package.json

```diff
@@ -5,9 +5,9 @@
   "license": "MIT",
   "dependencies": {
     "@0x/subproviders": "^6.0.8",
-    "@keep-network/keep-core": ">1.2.4-rc <1.2.4",
-    "@keep-network/keep-ecdsa": ">1.1.2-rc <1.1.2",
-    "@keep-network/tbtc": ">1.0.2-rc <1.0.2",
+    "@keep-network/keep-core": ">1.3.0-rc <1.3.0",
+    "@keep-network/keep-ecdsa": ">1.2.0-rc <1.2.0",
+    "@keep-network/tbtc": ">1.1.0-rc <1.1.0",
```

Comment:
These bumps are in the RC branches, which have now landed on master.

Reply:
I've done an upstream sync. Resolving.